include/tvm/runtime/crt/rpc_common/write_stream.h

```diff
@@ -29,6 +29,7 @@
 #include <stddef.h>
 #include <sys/types.h>
 #include <tvm/runtime/crt/error_codes.h>
+#include <tvm/support/ssize.h>
```

Comment:
@mkatanbaf @areusch I moved ssize.h to this location as otherwise it gets more complicated to include it in other header files, especially across different build flavors (linux, windows, etc.), let me know what you think.

Reply:
Thank you @alanmacd! LGTM, I can't think of any reason why this would be a problem.